Iteration 1
(node:4592) UnhandledPromiseRejectionWarning: TypeError: scripts.getTTLdTableItems is not a function
    at main (/home/ubuntu/realworld-dynamodb-lambda/analyze-ddb-use.js:3:38)
    at Object.<anonymous> (/home/ubuntu/realworld-dynamodb-lambda/analyze-ddb-use.js:8:1)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Function.Module.runMain (module.js:693:10)
    at startup (bootstrap_node.js:188:16)
    at bootstrap_node.js:609:3
(node:4592) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:4592) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
